xtas/tasks/_nl_conll_ner.py

```diff
@@ -31,12 +31,12 @@ def _download_training_data():
 
     Returns an iterable over the lines of the concatenated dataset.
     """
-    return (ln for part in ["train", "testa", "testb"]
+    return (ln for part in ["train", "testa", "testb"] #
                for ln in urlopen(_BASE_URL + part))
 
 
 
-def _features(sentence, i):
```

Comment:
If seqlearn let you return a list rather than an iterable, this function would look a bit nicer IMHO. It's only ever going to be a small list of features I think, so I can't imagine performance being an issue?